x-pack/test/functional/services/machine_learning/api.ts

```diff
@@ -388,6 +388,17 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
       await this.waitForDataFrameAnalyticsJobToExist(analyticsId);
     },
 
+    async startDataFrameAnalyticsJob(analyticsId: string) {
```

Comment:
Not sure if we want to keep since it's not called anywhere.
But if we keep it: This file has been moved as part of #67312, so you should merge upstream and apply the changes to the moved file.

Reply:
I think we can take this out and use it in another PR if needed in the future.